src/Components/CloudAccounts/__tests__/GetStatusIcon.test.tsx

```diff
@@ -0,0 +1,33 @@
+import { render, screen } from '@testing-library/react';
+import { getStatusIcon } from '../GetStatusIcon';
+
+describe('getStatusIcon', () => {
+  it('returns green check icon for Granted', () => {
+    render(getStatusIcon('Granted', 'Granted'));
+    const icon = screen.getByLabelText('Granted');
+
+    expect(icon).toBeInTheDocument();
+    expect(icon).toHaveAttribute('color', 'green');
+  });
+
+  it('returns red exclamation icon for Failed', () => {
+    render(getStatusIcon('Failed', 'Failed'));
+    const icon = screen.getByLabelText('Failed');
+
+    expect(icon).toBeInTheDocument();
+    expect(icon).toHaveAttribute('color', 'red');
+  });
+
+  it('returns black wrench icon for Requested', () => {
+    render(getStatusIcon('Requested', 'Requested'));
+    const icon = screen.getByLabelText('Requested');
+
+    expect(icon).toBeInTheDocument();
+    expect(icon).toHaveAttribute('color', 'black');
```

Comment:
Issue: These three tests are not actually testing _what_ icon is rendered, just that one is